### ISSUE TITLE
Jabref doesn't quit

### ISSUE THREAD

**Issue by paboe** [JabRef/jabref#11483]
### JabRef version

5.15 (latest release)

### Operating system

macOS

### Details on version and operating system

macOS Sonoma 14.5, JabRef 5.15, Java 1.8.0_371

### Checked with the latest development build (copy version output from About dialog)

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. Open a Library
2. Select "JabRef -> Quit" in the Menu bar or use the hot key "Cmd + Q"
3. The Window closes, but the program is still running.

JabRef will only quit if you force quit it.

**Comment by koppor** [comment]
Does the library have special properties? Like more than 1000 entries?

Do you have fulltext search enabled?

Would you feel comfortable in installing https://visualvm.github.io/ and checking the "Threads" of the JabRef Launcher?

This refs https://github.com/InAnYan/jabref/issues/77

**Comment by paboe** [comment]
It is a small library with no special properties. There is no file attached to any of the entries, so full text search doesn't apply. 

VisualVM shows me 29 Live Threads and 17 Daemon Threads.

**Comment by ra1nb0w** [comment]
Same issue with Linux. It happens with 6.0.0 downloaded from release and 5.15 built from source using JDK21.
In attachment the heap dump from visualvm of 6.0.0. Before starting the program, I cleared the configuration `.java/.userPrefs/org/` and `.local/share/jabref`.

https://www.dropbox.com/scl/fi/cqy5ss059d2mjpzzojedq/heapdump-1738012289260.hprof.zip?rlkey=jqyobpaanzwjnrd6hid4mw72o&st=pngizod9&dl=0

**Comment by ilippert** [comment]
Hi there,

is this problem still present? Cheers!

**Comment by ra1nb0w** [comment]
yes, In my case un-setting `DBUS_SESSION_BUS_ADDRESS='unix:path=/run/user/1000/bus'` permits the application to close.
Maybe, my Archlinux is not exposing something that the application requires since in Ubuntu 24.04 it works fine.
`---debug` is not helpful in this case.

**Comment by paboe** [comment]
In my case the problem is solved. The app quits as it should do.

**Comment by koppor** [comment]
Action: add documentation to the linux FAQ page.

**Comment by Siedlerchr** [comment]
Does this issue still occur in the latest dev version?  https://builds.jabref.org/main/
